js/ui/components/clubhouse.js

```diff
@@ -135,12 +135,6 @@ var ClubhouseWindowTracker = GObject.registerClass({
 
         this.emit('window-changed');
 
-        // Track Clubhouse's window actor to make the closeButton always show on bottom of
-        // the Clubhouse's window edge
-        this._notifyHandler = actor.connect('notify::y', () => {
```

Comment:
We have to remove it as well from _untrackWindowActor:

`this._windowActor.disconnect(this._notifyHandler);`
`this._notifyHandler = 0;`

Reply:
Ok, I added this to the commit with git rebase.